features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/components/RoomListTopBar.kt

```diff
@@ -149,130 +150,131 @@ private fun DefaultRoomListTopBar(
     val statusBarPadding = with(LocalDensity.current) { WindowInsets.statusBars.getTop(this).toDp() }
 
     Box(modifier = modifier) {
-        MediumTopAppBar(
-            modifier = Modifier
-                .onSizeChanged {
-                    appBarHeight = it.height
-                }
-                .nestedScroll(scrollBehavior.nestedScrollConnection)
-                .avatarBloom(
-                    avatarData = avatarData,
-                    background = if (ElementTheme.isLightTheme) {
-                        // Workaround to display a very subtle bloom for avatars with very soft colors
-                        Color(0xFFF9F9F9)
-                    } else {
-                        ElementTheme.materialColors.background
-                    },
-                    blurSize = DpSize(avatarBloomSize, avatarBloomSize),
-                    offset = DpOffset(24.dp, 24.dp + statusBarPadding),
-                    clipToSize = if (appBarHeight > 0) DpSize(
-                        avatarBloomSize,
-                        appBarHeight.toDp()
-                    ) else DpSize.Unspecified,
-                    bottomSoftEdgeColor = ElementTheme.materialColors.background,
-                    bottomSoftEdgeAlpha = 1f - collapsedFraction,
-                    alpha = if (areSearchResultsDisplayed) 0f else 1f,
-                )
-                .statusBarsPadding(),
-            colors = TopAppBarDefaults.mediumTopAppBarColors(
-                containerColor = Color.Transparent,
-                scrolledContainerColor = Color.Transparent,
-            ),
-            title = {
-                val fontStyle = if (scrollBehavior.state.collapsedFraction > 0.5)
-                    ElementTheme.typography.aliasScreenTitle
-                else
-                    ElementTheme.typography.fontHeadingLgBold.copy(
-                        // Due to a limitation of MediumTopAppBar, and to avoid the text to be truncated,
-                        // ensure that the font size will never be bigger than 28.dp.
-                        fontSize = 28.dp.applyScaleDown().toSp()
+        val smallTextStyle = ElementTheme.typography.aliasScreenTitle
+        val largeTextStyle = ElementTheme.typography.fontHeadingLgBold.copy(
```

Comment:
Neat: Maybe rename the 2 val for something with more semantic, for instance:
```kotlin
        val collapsedTextStyle = ElementTheme.typography.aliasScreenTitle
        val expandedTextStyle = ElementTheme.typography.fontHeadingLgBold.copy(
```

Reply:
I was trying to follow the nomenclature used in `TwoRowsTopAppBar`, but I can see how this can be confusing without more context. I'll do the change, thanks!